doc/tech/approach.md

```diff
@@ -3,97 +3,226 @@
+
+The nature of a shared Underlay Network that shall ensure separation and be robust is that all code in the forwarding plane and in the control plane must be under the scrutiny and life cycle management of the HW Infrastructure Layer.
+
+This also imply that programmable forwarding functions in a Programmable Networking Fabric are shared resources and by that will have to get standardised interfaces over time to be useful for multi-vendor architectures such as ETSI NFV. Example of such future extensions of shared functionality implemented by a Programmable Networking Fabric could be L3 as a Service, Firewall as a Service and Load Balancing as a Service.
+
+> **Note:** Appliance-like applications that fully own its infrastructure layers (share nothing) could manage and utilize a Programmable Networking Fabric in many ways, but that is not a Cloud implementation and falls outside the use cases for these specifications.
 
 <a name="5.1.4"></a>
-### 5.1.4 Initial Logistical Steps
-
-> _**Editor's Note:** The purpose in this section is to communicate the responsibilities of the CNTT levels, as they pertain to the networking solution, and how they relate (i.e. what goes in each bucket). The initial content below needs to be expanded with examples of material representative of the scope for each bucket, and refined. It is expected this area be enhanced to help authors in various CNTT levels understand their purview, and ultimately be deleted._
-
-As with most CNTT subsystems, responsibility for Objectives, Requirements, Guidelines, etc. is divided among the CNTT constituencies, as follows:
-- **Tech:** High-level strategy and other informative writings
-  - e.g. Executive summary, considerations, intent, vision, etc.
-- **RM:** Generic modeling and abstraction
-  - e.g. Guidelines and other information applicable to all RAs
-  - Overall theory of operation
-- **RA-x:** architecture, APIs and other service level details
-- **RI-x:** networking information for the relevant lab or POD
-  - Considerations related to differences between target architecture (RM 5.1) and current RI implementation
-  - Factors specific to supporting RC
-  - Known non-conformances
-- **RC:** Details related to qualification
-  - Implications of testbed network environment vs. production environments
+### 5.1.4 Networking Reference Model
+
+The Networking Reference Model, based on the ETSI NFV model, is depicted in **Figure 5-1-4**. The Networking Reference Model describes the Container Virtualisation support and a strict separation of the HW Infrastructure and SW Infrastructure Layers in NFVI. It includes all above concepts and enables multiple well separated simultaneous SW Virtualisation domains allowing a mix of CaaS on Metal, CaaS on IaaS and IaaS on a shared HW infrastructure. 
```

Comment:
This figure mixes several ETSI terms with several terms completely unknown to ETSI, such as reference points in ETSI notation that don't exist in ETSI. This way of presenting it is confusing. It must be clear which parts are ETSI and which are not.
Also this is not only a Networking Reference Model, but includes compute and storage domain with similar additions to the ETSI architecture, without explaining which parts are added to ETSI terminology.

The Container Virtualization support as published by ETSI in IFA029 is not considered but different terms and building blocks included.

Reply:
@ulikleber: Agree, that is why we are saying, that  this reference model, is based on ETSI mode, as many other models in the industry.  I do not see any problem. Comparison with IFA029 will be one of the PR I want to raise.  As per you previous comments, ETSI scope is different, and hence we need a different model.

Reply:
Yes, let's go forward, but let's also find a way to make clearly visible what is part  of the ETSI architecture, and what is added to it, so people not familiar with it can easier understand.